### PR TITLE
added maps api

### PR DESCRIPTION
**Pull request by trinitydonohugh**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -38,12 +38,14 @@
...
-  
+
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
-    String name = getInput(request, "name-input", "");
-    String message = getInput(request, "message-input", "");
+    String name = request.getParameter("name-input");
+    String message = request.getParameter("message-input");
+    //String name = getInput(request, "name-input", "");
```
Comment (pengxianzhe) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: Unused code. Did you mean to remove them?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -73,12 +75,15 @@ private String getInput(HttpServletRequest request, String userInput, String def
...
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
-
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    Query query = new Query("Comment");
+
     PreparedQuery results = datastore.prepare(query);
+    
+    System.out.println("results");
```
Comment (pengxianzhe) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:84]: This print looks like test code.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -94,8 +99,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     Gson gson = new Gson();
+    String json = gson.toJson(comments);
+    json = gson.toJson(comments);
```
Comment (pengxianzhe) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:103]: Why do you need to convert comments to json twice?

File: portfolio/src/main/webapp/WEB-INF/appengine-web.xml [hunk 4]
```diff
@@ -7,4 +7,4 @@
     <!-- prevent unwanted caching when accessing via the web preview server -->
     <include path="/**" expiration="0s" />
   </static-files>
-</appengine-web-app>
+</appengine-web-app>
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/WEB-INF/appengine-web.xml:10]: nit: no newline at EOF

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (trinitydonohugh) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:103]: This was all done as a part of testing -- will remove them now and re-commit. Thank you!!

**Pull request merged**